src/ExtendedView.js

```diff
@@ -30,12 +30,12 @@ class ExtendedView extends React.Component {
               style={{backgroundImage: `url(${this.state.singleMovieData.backdrop_path})`, backgroundSize: "cover", height: "75vh"}}
             >
               <p className="title">{this.state.singleMovieData.title}</p>
-              <button className="rating">Rating: {((this.state.singleMovieData.average_rating/10)*100).toFixed(0)}%</button>
+              <div className="rating">Rating: {((this.state.singleMovieData.average_rating/10)*100).toFixed(0)}%</div>
```

Comment:
Div makes sense over button - nice! I wonder if a section here could be more semantic?